hack/lib/init.sh

```diff
@@ -66,7 +66,10 @@ storage.k8s.io/v1beta1\
 # not all group versions are exposed by the server.  This list contains those
 # which are not available so we don't generate clients or swagger for them
 KUBE_NONSERVER_GROUP_VERSIONS="
- imagepolicy.k8s.io/v1alpha1 
+ abac.authorization.kubernetes.io/v0 \
```

Comment:
You don't need the ending `\` unless you want the string to not contain newlines